packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -45,11 +46,9 @@ export interface AlarmWidgetProps extends MetricWidgetProps {
   readonly alarm: Alarm;
 
   /**
-   * Range of left Y axis
-   *
-   * @default 0..automatic
+   * Left Y axis
    */
-  readonly leftAxisRange?: YAxisRange;
+  readonly leftYAxis?: YAxis;
```

Comment:
This seems like a breaking change. Please update the PR description to describe it per the instructions in the contribution guide